server/controllers/UserController.js

```diff
@@ -116,7 +116,7 @@ class UserController {
           token: userToken,
         });
       })
-      .catch(err => (err.message));
+      .catch((err) => { console.log(err.message); });
```

Comment:
You forgot to remove the console.log on line 119. Please review the UserController.js for any other occurrence.  Thank you.